<launch>

  <arg name="robot" default="drone"/>

  <group ns="$(arg robot)">
  <!-- Image Processor Nodelet  -->
    <remap from="cam0/image_raw" to="cam0_image" />
    <remap from="cam1/image_raw" to="cam1_image" />
    <remap from="imu0" to="imu" />

    <node pkg="rovio" type="rovio_node" name="rovio" output="screen">
      <param name="camera0_config" value="$(find simcontrol)/config/cam0.yaml"/>
      <param name="camera1_config" value="$(find simcontrol)/config/cam1.yaml"/>

    </node>

    <node type="publish_img_imu.py" name="publish_script" pkg="simcontrol" />
  </group>

</launch>
